tests/testthat/test-estimate_KM.R

```diff
@@ -43,6 +43,9 @@
 #' T7.2 The function prefixes the function call with survival
 #' T8. Piped datasets still return accurate results
 #' T8.1 Piped datasets still return accurate results
+#' T9. The user can specify formula argument
+#' T9.1 The formula method returns the same results and the data method.
```

Comment:
rephrase as we dont work with methods anymore.

Reply:
updated

Reply:
I'll recheck later this week.